From 6ca3fcd2aec89b620792c3fbaba7f2c068219779 Mon Sep 17 00:00:00 2001
From: Pascal Paillet <p.paillet@st.com>
Date: Tue, 19 Mar 2024 17:38:26 +0100
Subject: [PATCH 0707/1695] dts: stm32: fix wake-up from Standby for stm32mp13
 boards

Fix wake-up from Standby for stm32mp135 boards.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git 6875b4a3790439f8b2e1417ff88c07f2114fd83c dts: stm32: fix wake-up from Standby for stm32mp13 boards]
Change-Id: I6685e73d2ce19a3f0060f68b372d1fff4ff314e7
Signed-off-by: Pascal Paillet <p.paillet@st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/optee/optee_os/+/367954
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
Domain-Review: Patrick DELAUNAY <patrick.delaunay@foss.st.com>
Reviewed-by: Gatien CHEVALLIER <gatien.chevallier@st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
---
 core/arch/arm/dts/stm32mp135f-dk.dts | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/core/arch/arm/dts/stm32mp135f-dk.dts b/core/arch/arm/dts/stm32mp135f-dk.dts
index 6ac51273b..e569d24c1 100644
--- a/core/arch/arm/dts/stm32mp135f-dk.dts
+++ b/core/arch/arm/dts/stm32mp135f-dk.dts
@@ -318,7 +318,7 @@
 				regulator-name = "vdd_sd";
 				regulator-min-microvolt = <3300000>;
 				regulator-max-microvolt = <3300000>;
-				regulator-boot-on;
+				st,regulator-warm-boot-on;
 
 				standby-ddr-sr {
 					regulator-off-in-suspend;
-- 
2.47.2

